e2e/pages/adf/settingsPage.ts

```diff
@@ -96,9 +101,63 @@ export class SettingsPage {
         return this;
     }
 
-    clickApply() {
+    async clickSsoRadioButton () {
+        Util.waitUntilElementIsVisible(this.ssoRadioButton);
+        await this.ssoRadioButton.click();
+    }
+
+    async setProviderBpmSso (processServiceURL, authHost, silentLogin, implicitFlow ) {
```

Comment:
Let's make silentLogin and implicitFlow optional. If they are not set, let's use default values for silentLogin and implicitFlow: true for both.